### PR TITLE
Rj/update version chart eol support and EM

### PR DESCRIPTION
**Pull request by neontty**
adding end of life support column to version chart and support for adding "extended maintenance" EM Versions (tracked in the script which generates the table. )

### CODE REVIEW

**Comment by neontty** [comment]
blocks https://github.com/SimpleDataLabsInc/prophecy/pull/14139

**Comment by neontty** [comment]
new logic is to support all versions for default by 6 months. EM versions supported for 1 year.

added requirements.txt for easier dependency tracking in prophecy mono repo which calls this script.

**Review by alexanderahn — APPROVED** [review]
Thanks!

File: scripts/find_versions_from_git_tags.py [hunk 1]
```diff
@@ -3,10 +3,14 @@
 import re
 import os
 from datetime import datetime
+from dateutil.relativedelta import relativedelta
 from packaging import version as packaging_version
 
 versions = []
 
+LTS_VERSIONS = [
+    # "v3.4.1.0"
```
Comment (neontty) [inline scripts/find_versions_from_git_tags.py:12]: @alexanderahn just FYI, any time you want to add a new EM release you'll have to add it to this list here (similar to the commented out line)

**Pull request merged**